dbt/models/iasworld/schema/iasworld.asmt_all.yml

```diff
@@ -57,6 +57,16 @@ sources:
                       AND valclass IS NULL
                   meta:
                     description: at least one class should match pardat class
+              - res_class_matches_pardat:
+                  name: iasworld_asmt_all_ex_matches_pardat_class_ex
+                  major_class_only: false
+                  join_type: inner
+                  additional_select_columns: *select-columns-aggregated
+                  additional_pardat_filter: AND class = 'EX'
```

Comment:
I took a look at each of the unique conditions. As far as I understood, none interacted with 'Ex', but that is something to keep in mind.